compiler/codegen/CodeGenRA.cpp

```diff
@@ -556,9 +568,9 @@ OMR::CodeGenerator::allocateSpill(int32_t dataSize, bool containsCollectedRefere
 void
 OMR::CodeGenerator::freeSpill(TR_BackingStore *spill, int32_t dataSize, int32_t offset)
    {
-   TR_ASSERT(1 <= dataSize && dataSize <= 16, "assertion failure");
-   TR_ASSERT(offset == 0 || offset == 4, "assertion failure");
-   TR_ASSERT(dataSize + offset <= 16, "assertion failure");
+   TR_ASSERT_FATAL(1 <= dataSize && dataSize <= 64, "assertion failure");
+   TR_ASSERT_FATAL(offset == 0 || offset == 4, "assertion failure");
+   TR_ASSERT_FATAL(dataSize + offset <= 64, "assertion failure");
```

Comment:
You may want to take this opportunity to write a more informative assert message for each here.